dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java

```diff
@@ -158,6 +166,25 @@ public Tool(long id, String name) {
         workflowVersions = new TreeSet<>();
     }
 
+    @Override
+    public String getDefaultVersion() {
```

Comment:
Can you put the implementation in entry? Looks like the Tool and Workflow implementations are identical.